Sources/StreamVideoSwiftUI/CallViewModel.swift

```diff
@@ -586,6 +586,10 @@ open class CallViewModel: ObservableObject {
                         }
                     }
                 } else if let participantEvent = callEventsHandler.checkForParticipantEvents(from: event) {
+                    guard participants.count < 25 else {
```

Comment:
Are we completely skipping them? Would that have any unwanted effects on UI? (e.g. participants list count not updating)

Reply:
For now yes, because it is actually not usable nor correct. We need to re-think this (sth like "User and 10 more joined"), but not for the scope of this PR.